drivers/qcommsimmodem/radio-settings.c

```diff
@@ -76,6 +77,24 @@ static void qcom_msim_set_rat_cb(struct ril_msg *message, gpointer user_data)
 	}
 }
 
+static void qcom_msim_do_set_rat_mode(struct ofono_radio_settings *rs, int pref
+							, struct cb_data *cbd) {
```

Comment:
Do not split the line between a variable and a comma, and also that "{" should be starting a line.